### ISSUE TITLE
Uncaught TypeError: Cannot read property 'scanInBufferRange' of undefined

### ISSUE THREAD

**Issue by guzishiwo** [richrace/highlight-selected#127]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.8.0
**System**: Ubuntu 16.04
**Thrown From**: [highlight-selected](https://github.com/richrace/highlight-selected) package, v0.11.2
### Stack Trace

Uncaught TypeError: Cannot read property 'scanInBufferRange' of undefined

```
At /home/wei/.atom/packages/highlight-selected-master/lib/highlighted-area-view.coffee:130

TypeError: Cannot read property 'scanInBufferRange' of undefined
    at HighlightedAreaView.module.exports.HighlightedAreaView.highlightSelectionInEditor (/home/wei/.atom/packages/highlight-selected-master/lib/highlighted-area-view.coffee:130:11)
    at /home/wei/.atom/packages/highlight-selected-master/lib/highlighted-area-view.coffee:123:10
    at Array.forEach (native)
    at HighlightedAreaView.module.exports.HighlightedAreaView.handleSelection (/home/wei/.atom/packages/highlight-selected-master/lib/highlighted-area-view.coffee:122:27)
    at HighlightedAreaView.handleSelection (/home/wei/.atom/packages/highlight-selected-master/lib/highlighted-area-view.coffee:1:1)
    at /home/wei/.atom/packages/highlight-selected-master/lib/highlighted-area-view.coffee:49:8
```
### Commands

```
     -6:38.6.0 vim-mode-plus:activate-insert-mode (atom-text-editor.editor.vim-mode-plus.power-mode.is-focused.normal-mode)
     -6:38.3.0 editor:newline (atom-text-editor.editor.vim-mode-plus.power-mode.is-focused.insert-mode)
     -6:37.8.0 vim-mode-plus:activate-normal-mode (atom-text-editor.editor.vim-mode-plus.power-mode.is-focused.insert-mode)
     -6:37.1.0 vim-mode-plus:move-up (atom-text-editor.editor.vim-mode-plus.power-mode.is-focused.normal-mode)
     -6:36.7.0 vim-mode-plus:move-down (atom-text-editor.editor.vim-mode-plus.power-mode.is-focused.normal-mode)
     -6:36.3.0 vim-mode-plus:move-up (atom-text-editor.editor.vim-mode-plus.power-mode.is-focused.normal-mode)
  2x -6:35.9.0 vim-mode-plus:delete (atom-text-editor.editor.vim-mode-plus.power-mode.is-focused.normal-mode)
  2x -6:35.2.0 vim-mode-plus:move-right (atom-text-editor.editor.vim-mode-plus.power-mode.is-focused.normal-mode)
 11x -6:34.5.0 vim-mode-plus:move-to-next-word (atom-text-editor.editor.vim-mode-plus.power-mode.is-focused.normal-mode)
     -6:30.4.0 vim-mode-plus:move-up (atom-text-editor.editor.vim-mode-plus.power-mode.is-focused.normal-mode)
 13x -6:29.6.0 vim-mode-plus:move-left (atom-text-editor.editor.vim-mode-plus.power-mode.is-focused.normal-mode)
     -0:29.1.0 vim-mode-plus:activate-insert-mode (atom-text-editor.editor.vim-mode-plus.power-mode.normal-mode.is-focused)
     -0:28.6.0 editor:newline (atom-text-editor.editor.vim-mode-plus.power-mode.is-focused.insert-mode)
  3x -0:26.3.0 core:backspace (atom-text-editor.editor.vim-mode-plus.power-mode.is-focused.insert-mode)
     -0:23.4.0 markdown-preview:toggle (atom-text-editor.editor.vim-mode-plus.power-mode.is-focused.insert-mode)
  4x -0:02.4.0 core:backspace (atom-text-editor.editor.vim-mode-plus.power-mode.insert-mode.is-focused)
```
### Config

``` json
{
  "core": {
    "disabledPackages": [
      "atom-terminal",
      "atom-miku"
    ],
    "ignoredNames": [
      ".git",
      ".hg",
      ".svn",
      ".DS_Store",
      "._*",
      "Thumbs.db",
      "*.pyc"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
activate-power-mode, v0.5.2
autocomplete-python, v1.8.1
file-icons, v1.7.15
linter, v1.11.10
linter-flake8, v1.13.2
markdown-scroll-sync, v2.1.2
minimap, v4.24.6
pretty-json, v1.4.1
python-indent, v0.4.3
python-tools, v0.6.7
regex-railroad-diagram, v0.16.0
rest-client, v1.1.1
terminal-plus, v0.14.5

# Dev
No dev packages
```

**Comment by richrace** [comment]
Is this checkout from the git repo or installed via Atom?

**Comment by unixc3t** [comment]
@richrace 
I also encountered this problem

1 I git clone this in ~/.atom/packages/
2 i use apm install this

**Comment by richrace** [comment]
The master of this branch is not the published version.

Please checkout this commit (latest published) 0b192037afd7a364efc858da2661e7bbc2d7354e

**Comment by richrace** [comment]
Please update this package to the latest (0.13.0)
